CHANGELOG.md

```diff
@@ -21,6 +22,66 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v0.6.1
+
+This is a patch release that predominantly includes updated conformance tests
+for implementations to implement.
+
+For all major changes since the `v0.5.x` release series, please see the
+[v0.6.0](/#v060) release notes.
+
+## Bug Fixes
+
+- Our regex for validating path characters was updated to accurately identify
+  "p-chars" as per RFC-3986.
+  (#1644, @jackstine)
+- An erroneous "namespace" field was present in our webhook ClusterRoleBindings
+  and has been removed.
+  (#1684, @tao12345666333)
+
+## New Features
+
+- Conditions for Policies have been added to the Golang library, enabling
+  Go-based implementations to re-use those for their downstream Policies.
+  (#1682, @mmamczur)
+
+## Conformance Test Updates
+
+- Added conformance tests for checking Port, Scheme and Path to the extended and
+  experimental features.
+  (#1611, @LiorLieberman)
+- Added conformance tests for HTTP rewrite
+  (#1622, #1628, @LiorLieberman)
+- Added more conformance tests for path matching to catch known edge cases.
+  (#1627, @sunjayBhatia)
+- Added some initial conformance tests for TLSRoute passthrough.
+  (#1579, @candita)
+- Added conformance tests that exercise NotAllowedByListeners reason.
+  (#1669, @mlavacca)
+- Loosen the Accepted check in GatewayClass observed generation tests to
+  provide a more realistic test for implementations.
+  (#1655, @arkodg)
+- A "SkipTests" field has been added to accomodate implementations in
+  running subsets of the tests as needed, this can be particularly helpful
+  for new implementations that want to add conformance iteratively.
+  (#1578, @mlavacca)
+- ParentRefNotPermitted added to support situations where ReferenceGrant
+  is not allowed due to lack of a ReferenceGrant in the destination NS.
+  (#1672, @Sajiyah-Salat)
+- Fixed a broken test for GRPCRoute that caused an erronous failure.
```

Comment:
Noting https://github.com/kubernetes-sigs/gateway-api/pull/1692#discussion_r1099010300 here, would be good to get fixed before the release.

Reply:
Yep, thank you @skriss we'll hold on making sure that's sorted out.

Reply:
I've created https://github.com/kubernetes-sigs/gateway-api/issues/1703 and it will be considered a blocker for this PR.